andi/utils.py

```diff
@@ -0,0 +1,9 @@
+def as_class_names(cls_or_collection):
```

Comment:
I tried to add tests for this function, and remove except: statements (because they are too broad - we shouldn't be catching e.g. KeyboardError here), and the behavior of this function was not obvious. E.g.

```py
    >>> class Foo: pass
    >>> as_class_names(Foo)
    'Foo'
    >>> as_class_names([Foo])
    ['Foo']
    >>> as_class_names([Foo, str])
    ['Foo', 'str']
    >>> as_class_names([Foo, str, 123])
    "[<class 'andi.utils.Foo'>, <class 'str'>, 123]"
```

Note that in the last example `str(cls_or_collection)` is called. I actually liked the output of the last statement most, because not only `class.__name__` was available, but import path as well - name on its own can be ambiguous.

What do you think about removing this function altogether, and just converting to str when needed?

Reply:
Makes sense. I was trying to remove the boilerplate "<class ''> but clearly this function is not the right way.